tox.ini

```diff
@@ -1,25 +1,17 @@
 [tox]
-envlist = py3, validators
+# py37, py38 are predefined in tox so we don't need any extra configuration for them
+envlist = py37
```

Comment:
Will we be Able to run only pytest via tox? During development it would be nice to not run pylint and black on every invocation.

Reply:
Not via tox but if you work with pyenv, you should be able to do `pytest tests/` anytime. If you prefer I can split it back into 2 environments

Reply:
Yeah I know but i prefer keeping everything in virtualenv. So lets keep them separated. 🙂